gcs/pom.xml

```diff
@@ -208,10 +208,13 @@
               </filters>
               <artifactSet>
                 <includes>
+                  <include>com.fasterxml.jackson.core:jackson-core</include>
+                  <include>com.google.api</include>
                   <include>com.google.api-client</include>
                   <include>com.google.api.grpc</include>
                   <include>com.google.apis</include>
                   <include>com.google.auth</include>
+                  <include>com.google.cloud</include>
```

Comment:
What will this pull in?

Reply:
We need to include cloud-core lib as well, reverting to `com.google.cloud`